source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/data.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { withData as withOrbit, WithDataProps } from 'react-orbitjs';
+import { TYPE_NAME as NOTIFICATION, NotificationAttributes } from '@data/models/notification';
+import { query, defaultSourceOptions } from '@data';
+import { withStubbedDevData } from '@data/with-stubbed-dev-data';
+import { uuid } from '@orbit/utils';
+
+// const mapNetworkToProps = (passedProps) => {
+//   return {
+//     notifications: q => q.findRecords(NOTIFICATION)
+//   };
+// };
+
+const mapRecordsToProps = (passedProps) => {
+  return {
+    notifications: q => q.findRecords(NOTIFICATION)
+  };
+};
+
+export interface DataProps {
+  notifications: Array<JSONAPI<NotificationAttributes>>;
+  seenNotifications: Array<JSONAPI<NotificationAttributes>>;
+  showNotifications: Array<JSONAPI<NotificationAttributes>>;
+}
+
+export interface ActionProps {
+  markNotificationsToSeen: () => void;
+  clearAll: () => void;
+  clearOne: (id: string) => void;
+  markNotificationToSeen: (notification: JSONAPI<NotificationAttributes>) => void;
+}
+
+export function withData(WrappedComponent) {
+
+  class DataWrapper extends React.Component<DataProps & WithDataProps> {
+
+    markNotificationsToSeen = async () => {
+
+      const { notifications } = this.props;
+
+      await this.props.updateStore(t =>
+        notifications.map(not => t.replaceAttribute({
```

Comment:
`not` should be changed to `notification`. `not` means things. lol

Also, this call to updateStore should utilize `defaultOptions()`